CHANGELOG/CHANGELOG-0.4.md

```diff
@@ -4,11 +4,11 @@ Changes since `v0.3.0`:
 
 ### Features
 
+- Change the way job's frameworks are integrated into the controller manager. #695
```

Comment:
I would not include this in the release note. This is a cleanup, not a feature.

Reply:
done